pre-commit.Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Developer:
+# ---------
+# Name:      Maik Ellerbrock
```

Comment:
I wouldnt write this under Maik's name. as he did not write it, you can write it your name here, it does not bear any responsibility 😄